src/cmd-build

```diff
@@ -219,62 +218,35 @@ echo "New build ID: ${buildid}"
 
 imageprefix="${name:?}"-"${buildid}"
 # Make these two verbose
-set -x
-mkdir -p tmp/anaconda
+#set -x
+#mkdir -p tmp/anaconda
 img_base=tmp/${imageprefix}-base.qcow2
 
-# These options don't work for EL7 so don't pass for now
-# virt-install --console=log.file doesn't work for qemu in EL7
-# 9pfs isn't an option for EL7 so copying out anaconda logs doesn't work
-extraargs=
-if [ -n "${ISFEDORA}" ]; then
-    extraargs="${extraargs} --console-log-file ${PWD}/install.log --logs ${PWD}/tmp/anaconda"
-fi
+set -x
 
-if ${image_config}; then
-    extraargs="${extraargs} --image-config ${image_input}"
-else
-    extraargs="${extraargs} --kickstart ${image_input}"
-fi
+create_disk() {
+	dest="$1"
+	shift
 
-# forgive me for this sin
-iso_location="${workdir}/installer/$(basename "${INSTALLER}")"
-checksum_location="${workdir}/installer/$(basename "${INSTALLER_CHECKSUM}")"
+	rm -f "$(pwd)/diskimage.raw"
+	truncate -s 8G "$(pwd)/diskimage.raw"
```

Comment:
Let's get the size from `image.yaml`.

Reply:
Do we parse `image.yaml` into anything readable by bash anywhere?